x-pack/platform/plugins/shared/cases/public/api/utils.ts

```diff
@@ -63,6 +63,10 @@ export const convertToCamelCase = <T, U extends {}>(obj: T): U =>
 
 export const convertCaseToCamelCase = (theCase: Case): CaseUI => {
   const { comments, ...restCase } = theCase;
+  const settings = restCase.settings;
```

Comment:
I understand why you would do this here, but I would actually create it's own dedicated function for it rather than making the mutation a side effect of a separate function.

Technically, we shouldn't need this anymore with the backfill, but this is a safeguard?